src/test/java/uk/gov/ons/ctp/integration/ratelimiterclient/RateLimiterClient_IT.java

```diff
@@ -36,6 +38,7 @@
 
   private RestClient restClient;
   private RateLimiterClient client;
+  private CircuitBreaker circuitBreaker;
```

Comment:
In the javadoc for this class, can you please give an example of LIMITER_HOST  (eg "localhost") .

Maybe I'm stupid, but I tried "http://localhost:8181" , then "http://localhost" before I got the right one !

Reply:
Example added.